autogpt_platform/frontend/src/tests/auth.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { test, expect } from "./fixtures";
```

Comment:
Can we have some tests for invalid credentials, unlogged in users and admin login users too?

Reply:
I understand where you're coming from but scope of this ticket is very tight to good path items listed in linked issue